content/en/xtc/basics/045-organizations.md

```diff
@@ -51,3 +51,50 @@ In _Configuration_, you may edit the name, short name and description of your or
+{{% /note %}}
+
+When accessing an organization, the system will check whether the login requirements are fulfilled _in the current session_. Users that have an external login provider _and_ 2FA configured will need to use the appropriate login flow to access a specific organization. This can mean having to log out and log in again using a different authentication method in order to switch organizations.
+
+{{% warning notitle %}}
+Changes to the login requirement take effect immediately. Users that are already logged in will get the appropriate lock-out message on their next access to any of the organization's data. This includes saving any changes they may have made in the meantime. Admins *SHOULD* therefore try to schedule changes in a way to minimize user impact (e.g. outside of business hours or during an announced maintenance window).
+{{% /warning %}}
+
+{{% note notitle %}}
+XTC currently does not support requiring both, 2FA and SSO login. If your security policies require this setup please configure your SSO provider as the sole allowed login method and configure 2FA as a requirement in your provider's setup.
+{{% /note %}}
+
+#### Mandatory Two-Factor Authentication
+
+To enable mandatory 2FA for an organization go to the organization's _Configuration_, open the _Security_ tab and edit _Access Requirements_. The the value to _Two-Factor login required_ and save the changes.
+
+{{% warning notitle %}}
+If you are not currently logged in via 2FA the system will warn you about _locking yourself out of the organization_ (see example screenshot). Take this warning seriously as you will not be able to access the organization (and therefore change the login requirement) after saving the configuration!
+{{% /warning %}}
+
+{{< image src="xtc/login_requirement_2fa_violated.png" >}}
+Example of an admin about to lock themselves out of an organization by enabling mandatory 2FA.
+{{< /image >}}
+
+#### Mandatory Login via an External Provider
+
+To enable the mandatory use of an external login provider go to the organization's _Configuration_, open the _Security_ tab and edit _Access Requirements_. The the value to _SSO login required_ and save the changes.
+
+{{< image src="xtc/login_requirement_sso.png" >}}
+Mandatory External Login Provider with a specific issuer URL.
+{{< /image >}}
+
+Admins can allow a specific set of login providers by putting their issuer URLs into the text field. Issuer URLs are the exact values of the `iss` claim in the OpenID Connect access token signed by your provider (e.g. `https://accounts.google.com` for the public Google identity provider). The system supports multiple allowed issuers by putting each URL on a seperate line.
+
+{{% warning notitle %}}
```

Comment:
Feel free to put an additional title on this, like `{{% warning title="Accidental Lockout" %}}`, to put more emphasis on what might go wrong here (would be pretty stupid if it does).

Reply:
Fixed.